keygen

```diff
@@ -25,40 +46,51 @@ if [ "$OPENSSL_VERSION_MAJOR" -lt 1 ]; then
   exit 1
 fi
 
+if [ -z "$CERT_CN" ]; then
+    echo "Common name (CERT_CN) not set."
+    echo -n "Enter a certificate common name [docker.mender.io]:"
+    IFS='' read -r CERT_CN
+    if [ -z "$CERT_CN" ]; then
+        CERT_CN="docker.mender.io"
+    fi
+fi
+
+if [ -z "$CERT_SAN" ]; then
+    CERT_SAN="DNS:${CERT_CN},DNS:*.${CERT_CN}"
+    echo "Subject alternative name (CERT_SAN) not set using default:"
+    echo "    CERT_SAN=\"${CERT_SAN}\""
+fi
+
+CERT_VALID_DAYS="3650"
+FILE_NAME_PRIVATE_KEY="private.key"
+FILE_NAME_CERT="cert.crt"
 
 ROOTDIR=$(pwd)/keys-generated
 
-CERTDIR=$ROOTDIR/certs
+CERTDIR=$ROOTDIR/cert
```

Comment:
Any particular reason this needs to be changed? Especially since træfik uses `certs` in its path.

Reply:
Guess I got a little too enthusiastic about merging the demo certificates into a common one :smiley_cat: 
I don't really want to revert it though (unless there is a strong preference) as it's already consistent with all the compose files.